contrib/linux/actions/checks/check_processes.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
Unsure if we could use python 2 like that, considering the fact that platform should be able to run in both environments.
Eg. there might be the cases when py2 available, while py3 is not present in the system and vice versa.

Reply:
Oh sorry, I should have mentioned that in the original post. So I saw that we solved the issue with `python2` in other places, too. After some testing on Ubuntu 14.04-18.04 and CentOS6-8 I saw that all of them have python2 available per default and I also remember a short slack discussion on this topic. 

So I think this is not really any risk but should be in the back of our mind when we deprecate Python2. This is just another topic that'll be way easier to handle once we support Python3 only.

Anyway, I'm all ears if you have any better idea to evaluate or maybe even a solution that truly supports both Python versions. :)

Reply:
Besides of VM there might be also Docker environment running without py2 installation.
We just can't add more `py2`-only things, considering active development and aim towards the [Python 2 Deprecation](https://github.com/StackStorm/discussions/issues/40), otherwise it'll just make the migration harder for us.

Reply:
So, can we live with these broken actions as "known issues" in py2-Environments and support Py3 only, here? 

For me, this would be fine. I mean, in the end it was one of the team who noticed the issue(s) and we did not get any complaints from users yet. I know this is not the best from the users perspective but handling it as known issue may work for the meantime.